__tests__/hooks/usePrefersReducedMotion.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { renderHook } from '@testing-library/react-hooks';
+
+import { usePrefersReducedMotion } from '../../src/hooks/usePrefersReducedMotion';
+
```

Comment:
I don't necessarily have a great alternative or reason they should be removed, but these do feel a little bit like they're just testing browser and hooks functionality instead of our code 🤷

Reply:
I think they're still meaningful! The first test in the suite validates that the hook returns the _correct_ `boolean` for the given scenarios we're testing, i.e. that encountering `(prefers-reduced-motion: reduce)` results in `true` being returned to the caller and that `(prefers-reduced-motion: no-preference)` results in `false` (and not vice versa).

I agree the second test is more "Ok, `useEffect` can probably handle this just fine." But still good to verify the event listeners get attached and cleaned up on unmount and that we don't risk leaving a rogue event listener hanging around causing a memory leak.